backend/src/main/java/com/zzang/chongdae/auth/exception/AuthErrorCode.java

```diff
@@ -0,0 +1,25 @@
+package com.zzang.chongdae.auth.exception;
+
+import com.zzang.chongdae.global.exception.ErrorMessage;
+import com.zzang.chongdae.global.exception.ErrorResponse;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@AllArgsConstructor
+public enum AuthErrorCode implements ErrorResponse {
+
+    INVALID_TOKEN(HttpStatus.UNAUTHORIZED, "유효하지 않은 토큰입니다."),
+    EXPIRED_TOKEN(HttpStatus.UNAUTHORIZED, "만료된 토큰입니다."),
+    INVALID_PASSWORD(HttpStatus.NOT_FOUND, "가입되지 않은 회원입니다."),
```

Comment:
```suggestion
    INVALID_PASSWORD(BAD_REQUEST, "가입하지 않은 회원입니다."),
```
NOT_FOUND 대신 BAD_REQUEST를 쓰는 건 어떤가요?
그리고 에러 내용을 조금 바꿨습니다.

Reply:
현재는 카카오톡 로그인 시 인증정보로 ci값"만" 사용하고 있는데요, ci가 memberRepository에 없으면 가입하지 않은 회원이기에 NOT_FOUND 를 응답했습니다.

Reply:
NOT_FOUND는 우리 서비스 내부 외에도 물리적인 네트워크 부분에서 발생할 수도 있을만한 에러이기 때문에 BAD_REQUEST를 추천하였습니다. 추후 다시 의논해보죠~

Reply:
[refactor: ci값이 일치하지 않을경우 오류메시지 문구 변경](https://github.com/woowacourse-teams/2024-chongdae-market/pull/177/commits/a2fdecb2ab58c6f485bc17eaf97ca5f33fa6b05f)